### PR TITLE
Added ftp urls to be catched in urlize

### PR DESCRIPTION
**Pull request by choialdrichdy**
using #522 as an inspiration, i have added the 'ftp://' url to be accepted in urlize. Also, i have separated both the domain and schemes in a list.

### CODE REVIEW

File: jinja2/utils.py [hunk 1]
```diff
@@ -197,6 +197,8 @@ def urlize(text, trim_url_limit=None, nofollow=False, target=None):
 
     If target is not None, a target attribute will be added to the link.
     """
+    scheme = ['http://','https://','ftp://']
```
Comment (ThiefMaster) [inline jinja2/utils.py]: PEP8 - there should be a space after `,`

File: jinja2/utils.py [hunk 2]
```diff
@@ -197,6 +197,8 @@ def urlize(text, trim_url_limit=None, nofollow=False, target=None):
...
+    domain = ['.com','.org','.net']
```
Comment (ThiefMaster) [inline jinja2/utils.py]: same

File: jinja2/utils.py (hunk 1 above)
Comment (ThiefMaster) [inline jinja2/utils.py]: `schemes`?

File: jinja2/utils.py (hunk 2 above)
Comment (ThiefMaster) [inline jinja2/utils.py]: `domains`?

Comment (ThiefMaster) [inline jinja2/utils.py]: Why not make those two lists tuples here instead of converting them later?

File: tests/test_utils.py [hunk 3]
```diff
@@ -81,7 +81,9 @@ def test_markup_leaks(self):
...
         url = "http://example.org"
+        url2 = "ftp://example.com"
         target = "<script>"
-        assert urlize(url, target=target) == ('<a href="http://example.org"'
-                                              ' target="&lt;script&gt;">'
-                                              'http://example.org</a>')
+        result = urlize(url2, target=target)
+        assert result == ('<a href="ftp://example.com"'
+                                              ' target="<script>">'
+                                              'ftp://example.com</a>')
```
Comment (ThiefMaster) [inline tests/test_utils.py]: you removed the file's trailing linebreak

**Comment by ThiefMaster** [comment]
Commit messages should use present tense / command style, e.g,. "Add ftp urls ..." instead of "Added ...". Also, you could squash the second commit into the first one - the fact that you fixed the tests in the first commit is not really something useful in the project history in case the PR gets merged.

**Comment by ThiefMaster** [comment]
Would it make sense to have only http/https by default and add an `extra_schemes` argument if you want to include stuff like ftp? I think ftp URLs are not that common after all.

**Comment by mitsuhiko** [comment]
We recently added a policy configuration. I'm happy to accept a policy for the url schemes supported. I'm not super happy with the urlize filter as it stands right now because it's not super correct. Instead of piling up on it i rather accept a more generalized PR that improves on it (eg: tld handling etc.).

**Pull request closed without merging**